subsys/net/lib/fota_download/src/util/fota_download_full_modem.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <nrf_modem.h>
+#include <modem/nrf_modem_lib.h>
+#include <zephyr/drivers/flash.h>
+#include <dfu/dfu_target_full_modem.h>
+#include <nrf_modem_bootloader.h>
+#include <dfu/fmfu_fdev.h>
+#include <string.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(fota_download_full_modem, CONFIG_LOG_DEFAULT_LEVEL);
+
+#define FULLMFU_BUF_SIZE (0x1000)
```

Comment:
Why not configurable through kconfig?

Reply:
Even if its not configurable, IMO its still preferable to have values like these in kconfig.

Reply:
I added kconfig for this.